modules/candidate_parameters/ajax/formHandler.php

```diff
@@ -136,21 +136,22 @@ function editProbandInfoFields($db, $user)
         header("HTTP/1.1 403 Forbidden");
         exit;
     }
-
-    $candID = $_POST['candID'];
+    //Sanitizing the post data
+    $sanitize = array_map('htmlentities', $_POST);
+    $candID   = $sanitize['candID'];
 
     // Process posted data
-    $gender = isset($_POST['ProbandGender']) ? $_POST['ProbandGender'] : null;
-    $dob    = isset($_POST['ProbandDoB']) ? $_POST['ProbandDoB'] : null;
+    $gender = isset($sanitize['ProbandGender']) ? $sanitize['ProbandGender'] : null;
```

Comment:
Could you write these lines instead as `$gender = sanitize['ProbandGender'] ?? null;`?

Reply:
@johnsaigle done!